modules/apps/knowledge-base/knowledge-base-api/src/main/java/com/liferay/knowledge/base/constants/KBConstants.java

```diff
@@ -27,4 +27,8 @@ public class KBConstants {
 
 	public static final String SERVICE_NAME = "com.liferay.knowledge.base";
 
+	public static final String TYPE_ARTICLE = "article";
+
+	public static final String TYPE_FOLDER = "folder";
```

Comment:
Can we use the KBArticle/KBFolder class names instead of made up constants?

Reply:
The thing is that they should match in [frontend side](https://github.com/liferay-lima/liferay-portal/blob/eb25bc63edca2058c887b1c93fefebfcaf00a490/modules/apps/knowledge-base/knowledge-base-web/src/main/resources/META-INF/resources/admin/js/KBDropdownPropsTransformer.js#L29), and here doesn't make much sense to work with java classnames. Is it a problem to have this constants?

Reply:
The problem is that now we have two things in the code that serve the same purpose but in different contexts. Maybe we could use `KBArticle.class.getSimpleName()` instead of these constants? The value of that will be `KBArticle` and `KBFolder` so the frontend won't have to deal with wild constants, and we keep some symmetry in the backend.